app/gui2/src/composables/doubleClick.ts

```diff
@@ -14,6 +14,7 @@ export function useDoubleClick<Args extends any[]>(
 
   const handleClick = (...args: Args) => {
     clickCount++
+    console.log(clickCount)
```

Comment:
stray log? or maybe hide it behind a debug setting... idk if we have a debug setting though.
we could add a boolean in `util/debug.ts`, or add a define (or multiple) that takes the value from an environment variable (or multiple) so we can do `DEBUG=true npm run dev` or `DEBUG=doubleClick,ls npm run dev`